src/utils/testUtils.js

```diff
@@ -32,3 +33,13 @@ export const hasRoleAttribute = (el, roleName) => {
 export const componentHasProperty = (component, prop, value) => {
 	expect(component && component.props && component.props[prop] === value).toBe(true);
 };
+
+// Note: functional components need to be
+// wrapped it in a stateful component to use TestUtils effectively
+export const TestWrapper = createReactClass({
+	render: function() {
```

Comment:
sure no problem. steve and I are on the team. I can do some reading - I just would like to do this upgrade/changes in a separate ticket to make sure it corrects this problem. I'm also checking in with adam to make sure he knows.